### ISSUE TITLE
Video size limited to 100MB

### ISSUE THREAD

**Issue by msn444** [Bostwickenator/STGUploader#17]
Any reason the limit can't be significantly higher?

**Comment by Bostwickenator** [comment]
Yes the Picasa API is terrible. New Google Photos API will fix this.

**Comment by Deph0** [comment]
It should now be possible to upload bigger files, however as the quote below suggest, it WILL count towards your storage capacity of your Google Account.

> Note: All media items uploaded to Google Photos through the API are stored in full resolution at original quality. If your uploads exceed 25MB per user, your application should remind the user that these uploads will count towards storage in their Google Account.
(src [API limits guide)](https://developers.google.com/photos/library/guides/api-limits-quotas) & [upload-media guide](https://developers.google.com/photos/library/guides/upload-media)
